### PR TITLE
Update model_license_other.md

### PR DESCRIPTION
**Pull request by thanktua-spp**
Hi guys!

I opened my first PRs for #985 - Just want to be sure I did that 1 right, I will be updating the model card license for a lot more models now that I have broken the ice.

### CODE REVIEW

File: hacktoberfest_challenges/model_license_other.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ Here is the list of models with `license:other` and a LICENSE file in the repo.
...
+|Opened|https://huggingface.co/decapoda-research/llama-7b-hf/discussions/125|[decapoda-research/llama-7b-hf](https://huggingface.co/decapoda-research/llama-7b-hf)|379224      |1223                     |llama-license                                                                                 |https://huggingface.co/decapoda-research/llama-7b-hf/blob/main/LICENSE                        |[LICENSE](https://huggingface.co/decapoda-research/llama-7b-hf/blob/main/LICENSE)                  |                                                                                                                     |                                                                                   |
```
[the remark this answers is not in the document]
Comment (thanktua-spp) [reply hacktoberfest_challenges/model_license_other.md:76]: thanks so much, @Wauplin this was very helpful feedback. Many thanks.

**Comment by Wauplin** [comment]
Thanks for making the change @thanktua-spp! Looks good to me like this. I've resolved the merge conflict and will now merge this PR. Well done!

**Comment by thanktua-spp** [comment]
thanks so much for supporting @Wauplin, I will go ahead and make some more updates. BTW are there any other Hectoberfest challenges to look forward to? I am already feeling confident 🤗

**Review by Wauplin — APPROVED** [review]

**Comment by Wauplin** [comment]
And... merged!

> I will go ahead and make some more updates. BTW are there any other Hectoberfest challenges to look forward to?

Yes! So for now we have three challenges related to metadata on the Hub (see https://github.com/huggingface/hub-docs/issues/985, https://github.com/huggingface/hub-docs/issues/986 and https://github.com/huggingface/hub-docs/issues/995). And we are thinking about releasing new ones during the Hacktoberfest!
Besides, any issues tackled in `huggingface_hub`, `transformers` or `diffusers` repositories are counted as valid for the Hacktoberfest. We've curated lists of _"Good first issue"_ that can be tackled by anyone. Some are more technical, some are more focused on documentation, etc. Check out the contribution guide, choose an issue and start right away! :)  ([huggingface_hub](https://github.com/huggingface/huggingface_hub/contribute), [transformers](https://github.com/huggingface/transformers/contribute), [diffusers](https://github.com/huggingface/diffusers/contribute)). Feel free to ping a maintainer if you have specific questions on an issue :hugs:

**Comment by thanktua-spp** [comment]
thanks so much for sharing @Wauplin.

**Pull request merged**